source/debug.rst

```diff
@@ -3,6 +3,38 @@
 Debug
 =================
 
+.. _debug-images-logs:
+
+Cluster-level logging
+---------------------
+
+Cluster-level logging involves collecting logs from all components of the
```

Comment:
For now we use it only for pxc pods and we use the same PVC as for pxc pod. The logs can be found in DATADIR /(var/lib/mysql/)